build/Jenkinsfile

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent none
+
+    environment {
+        MATTERMOST_DESKTOP_WINDOWS_PFX_PASSWORD = credentials('mattermost-desktop-windows-pfx-password')
+    }
+    stages {
+        stage('Build Linux') {
+            agent any
+            steps {
+                echo 'Building Mattermost Desktop App'
+                sh 'rm -rf release'
+                sh 'npm install'
+                sh 'npm run build'
+
+                echo 'Packaging for Linux'
+                sh 'npm run package:linux'
+                sh 'mv release/mattermost-desktop-*-ia32.tar.gz  release/tmp-linux-ia32.tar.gz'
+                sh 'mv release/mattermost-desktop-*.tar.gz release/mattermost-desktop-linux-x64.tar.gz'
+                sh 'mv release/tmp-linux-ia32.tar.gz release/mattermost-desktop-linux-ia32.tar.gz'
+                sh 'mv release/mattermost-desktop_*_i386.deb release/mattermost-desktop-linux-i386.deb'
+                sh 'mv release/mattermost-desktop_*_amd64.deb  release/mattermost-desktop-linux-amd64.deb'
```

Comment:
Like I said in the description of this PR, we might consider removing these `mv` statements and archiving the artifacts with version numbers in their filenames. That could help to keep artifacts sorted during release testing. We'd just have to manually rename them when uploading them to S3 once the release is ready. We could create another Jenkins pipeline to do that too if desired.